### PR TITLE
send SIGKILL on loginctl kill-session

### PR DESCRIPTION
**Pull request by umezawatakeshi**
- send SIGKILL on `loginctl kill-session`
- ~kill orphaned bash processes (whose PPID is 1)~

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: debian/usr/sbin/kill-old-login-sessions [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
   if [ $diff -gt $limit ]; then
     echo "kill session:"
     loginctl show-session $session
     echo
-    loginctl kill-session $session
+    loginctl kill-session --signal 9 $session
+  fi
+done
+
+# kill orphaned bash
```
Comment (ymmt2005) [inline debian/usr/sbin/kill-old-login-sessions:20]: @umezawatakeshi 
Have you seen orphaned bash processes after `loginctl kill-session --signal 9`?

Comment (umezawatakeshi) [reply debian/usr/sbin/kill-old-login-sessions:20]: As I test now, orphaned bash processes are not seen. So, this block is not necessary.

Some existing orphaned bash processes on some boot servers should be deleted manually when we enable pam_teleport.

Comment (ymmt2005) [reply debian/usr/sbin/kill-old-login-sessions]: I see.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**